vault/resource_github_auth_backend_test.go

```diff
@@ -0,0 +1,277 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccGithubAuthBackend_basic(t *testing.T) {
```

Comment:
When I attempt to run this test, I get the following failures. I'm sure at least some of them would be resolved by simply merging in the terraform vault provider's master branch.
```
./resource_aws_secret_backend_role.go:48:5: cannot use util.JsonDiffSuppress (type func(string, string, string, *"github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/schema".ResourceData) bool) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/schema".SchemaDiffSuppressFunc in field value
./structures.go:109:10: undefined: util.ShortDur
./structures.go:111:10: undefined: util.ShortDur
./structures.go:114:11: undefined: util.ShortDur
./resource_aws_secret_backend_role_test.go:33:36: cannot use util.TestCheckResourceAttrJSON("vault_aws_secret_backend_role.test_policy_inline", "policy", testAccAWSSecretBackendRolePolicyInline_basic) (type "github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc in argument to "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".ComposeTestCheckFunc
./resource_aws_secret_backend_role_test.go:44:36: cannot use util.TestCheckResourceAttrJSON("vault_aws_secret_backend_role.test_policy_inline", "policy", testAccAWSSecretBackendRolePolicyInline_updated) (type "github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc in argument to "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".ComposeTestCheckFunc
./resource_aws_secret_backend_role_test.go:68:36: cannot use util.TestCheckResourceAttrJSON("vault_aws_secret_backend_role.test_policy_inline", "policy", testAccAWSSecretBackendRolePolicyInline_basic) (type "github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc in argument to "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".ComposeTestCheckFunc
./resource_aws_secret_backend_role_test.go:102:36: cannot use util.TestCheckResourceAttrJSON("vault_aws_secret_backend_role.test_policy_inline", "policy", testAccAWSSecretBackendRolePolicyInline_basic) (type "github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc in argument to "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".ComposeTestCheckFunc
./resource_aws_secret_backend_role_test.go:113:36: cannot use util.TestCheckResourceAttrJSON("vault_aws_secret_backend_role.test_policy_inline", "policy", testAccAWSSecretBackendRolePolicyInline_updated) (type "github.com/terraform-providers/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc) as type "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".TestCheckFunc in argument to "github.com/cvbarros/terraform-provider-vault/vendor/github.com/hashicorp/terraform/helper/resource".ComposeTestCheckFunc
```